neuralpredictors/layers/shifters/mlp.py

```diff
@@ -0,0 +1,69 @@
+from .base import Shifter
+
+from torch.nn.init import xavier_normal
+from torch.nn import ModuleDict
+from torch import nn
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MLP(Shifter):
+    def __init__(self, input_features=2, hidden_channels=10, shift_layers=1, **kwargs):
+        """
+        Multi-layer perceptron shifter
+        Args:
+            input_features (int): number of input features, defaults to 2.
+            hidden_channels (int): number of hidden units.
+            shift_layers(int): number of hidden layers.
```

Comment:
I don't think this is an accurate description. Shift layers is literally the number of the layers including the hidden but also the mandatory output layer.